plutus-ledger-api/src/PlutusLedgerApi/V1/Bytes.hs

```diff
@@ -76,12 +78,17 @@ bytes = P.fromBuiltin . getLedgerBytes
 fromBytes :: BS.ByteString -> LedgerBytes
 fromBytes = LedgerBytes . P.toBuiltin
 
+{- | The `IsString` instance of `LedgerBytes` could throw an exception of `LedgerBytesError`. -}
 instance IsString LedgerBytes where
     fromString = unsafeFromEither . fromHex . fromString
 
+{- | The `Show` instance of `LedgerBytes` is its base16/hex encoded bytestring,
+decoded with UTF-8, unpacked to `String`. -}
 instance Show LedgerBytes where
     show = Text.unpack . encodeByteString . bytes
 
+{- | Encode a ByteString value in base16 (i.e. hexadecimal), then
+decode with UTF-8 to a `Text`. -}
```

Comment:
I have a feeling that going through Text and decodeUTF8 is redundant for showing base16 bytestrings.

Also do you know why we prepend '0x' only for tokennames? https://github.com/input-output-hk/plutus/blob/master/plutus-ledger-api/src/PlutusLedgerApi/V1/Value.hs#L115-L116

Reply:
Maybe it's redundant, since `asBase16` is only used to convert to `String` it seems: https://github.com/input-output-hk/plutus/blob/15b9c416ededcd7184db3c4f741737cadb194966/plutus-ledger-api/src/PlutusLedgerApi/V1/Value.hs#L121-L125 Not sure if I should be fixing code in this PR though. I think we should do it in the clean up PR? There's a ticket to clean up `plutus-ledger-api`.

I have no idea why we prepend '0x' only for token names.

Reply:
Ah no, it was just a general comment, not to be fixed by this PR (if it needs fixing)